README.md

```diff
@@ -5,6 +5,13 @@ to use our drone, simply add to your `Cargo.toml`, under `[dependencies]`:
 ``` rust
 null-pointer-drone = { git = "https://github.com/The-Null-Pointer-Patrol/null-pointer-drone.git"}
 ```
+Once you've done that you can declare :
+``` rust 
+use null_pointer_drone::MyDrone; 
+```
+# After-sales service
+if you encounter any problem with our drone you can open an issue on our github at [this link](https://github.com/The-Null-Pointer-Patrol/null-pointer-drone/issues)
+
```

Comment:
Here you could create a telegram group for support and put a qr code that points to it as @jarkko793 said

Reply:
maybe another pull request for that and if I have to create a group, I don't really want to go on telegram. I haven't any habits on this plateform and I'm not sure I can easily manage messages.